actioncable/lib/action_cable/test_helper.rb

```diff
@@ -0,0 +1,124 @@
+module ActionCable
+  # Provides helper methods for testing Action Cable broadcasting
+  module TestHelper
+    extend ActiveSupport::Concern
+
+    included do
+      def before_setup # :nodoc:
+        server = ActionCable.server
+        test_adapter = ActionCable::SubscriptionAdapter::Test.new(server)
+
+        @old_pubsub_adapter = server.pubsub
+
+        server.instance_variable_set(:@pubsub, test_adapter)
+        super
+      end
+
+      def after_teardown # :nodoc:
+        super
+        ActionCable.server.instance_variable_set(:@pubsub, @old_pubsub_adapter)
+      end
```

Comment:
First of all, to use _fresh_ server for every example. Of course, we can just use kind of `ActionCable.server.reset!` in `before_setup`.

Secondly, to make sure that we use test adapter and to allow users to use their preferred adapter in other test cases (e.g. acceptance testing).
Maybe it's better to provide some convenient helpers to switch between adapters and use "test" by default (from cable.yml).